src/styles/main.scss

```diff
@@ -1,3 +1,245 @@
+$Main-accent: #060b35;
+$Secondary: #616070;
+$Blue-accent: #00acdc;
+
 body {
   margin: 0;
+  line-height: 60px;
+  box-sizing: border-box;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-size: 12px;
+  font-weight: 500;
+  color: $Main-accent;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  background-color: white;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  text-transform: uppercase;
+}
+
+.main-navigation {
+  margin-right: 50px;
+
+  &__list {
+    height: 60px;
+    display: flex;
+    margin: 0;
+    line-height: 60px;
+    padding: 0;
+  }
+  &__item {
+    list-style: none;
+    text-transform: uppercase;
+    height: 60px;
+    display: flex;
+    flex-direction: column;
+    color: $Main-accent;
+  }
+
+  &__item:not(:first-of-type) {
+    margin-left: 20px;
+  }
+
+  &__item:first-of-type::after {
+    position: relative;
+    bottom: 4px;
+    content: "";
+    background-color: #00acdc;
+    height: 0;
+    padding: 2px 0;
+    border-radius: 2px;
+    width: 100%;
+  }
+
+  &__item > a {
+    text-decoration: none;
+    color: $Main-accent;
+  }
+
+  &__item > a:hover {
+    color: $Blue-accent;
+  }
+  .is-active {
+    color: $Blue-accent;
+  }
+}
+
+.logo {
+  align-self: center;
+  height: 40px;
+  width: 40px;
+  margin-left: 50px;
+}
+
+.main-page {
+  min-width: 944px;
+  width: 100%;
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+.container {
+  width: 944px;
+  height: 862px + 50px;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
```

Comment:
It is said in the readme that if you are familiar with grid it is mandatory to use it.
I'm aware of that you have done this task a while ago but I would like you to refactor your layout to grid. And additionally add media queries for [RWD](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Responsive_Design) - remember about mobile first approach.
Here are the [grid layout instrutions](https://github.com/mate-academy/layout_catalog/blob/master/grid.md) from the readme